modules/core/src/main/java/org/apache/synapse/config/xml/CallMediatorFactory.java

```diff
@@ -91,8 +93,15 @@ public Mediator createSpecificMediator(OMElement elem, Properties properties) {
                 callMediator.setEndpoint(endpoint);
             }
         }
-
+        // create the blocking attribute and set it in the call mediator
+        OMAttribute blocking = elem.getAttribute(BLOCKING_Q);
+        if (blocking != null) {
+            callMediator.setBlocking(Boolean.parseBoolean(blocking.getAttributeValue()));
+        } else {
+            callMediator.setBlocking(false);
```

Comment:
Yes,No need the else clause in this case.The following PR is already added with the modification.
https://github.com/wso2/wso2-synapse/pull/188